.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: r
+r:
+  - oldrel
```

Comment:
Testing against the old R package, i think it's set to 2.7.0: https://docs.travis-ci.com/user/languages/r